### PR TITLE
fly-206: remove apply when no changes.

### PR DESCRIPTION
**Pull request by russmack**
This PR implements https://github.com/concourse/fly/issues/206 - removing the "apply configuration" prompt when there are no changes.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@russmack Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/fly&pullRequestId=208)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=concourse/fly&pullRequestId=208) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@russmack Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=concourse/fly&pullRequestId=208)!

File: commands/internal/setpipelinehelpers/atc_config.go [hunk 1]
```diff
@@ -270,11 +275,15 @@ func (atcConfig ATCConfig) showHelpfulMessage(created bool, updated bool) {
 	}
 }
 
-func diff(existingConfig atc.Config, newConfig atc.Config) {
+func diff(existingConfig atc.Config, newConfig atc.Config) int {
```
Comment (vito) [inline commands/internal/setpipelinehelpers/atc_config.go]: This could probably just be a boolean return value.

**Comment by russmack** [comment]
Seems shashwathi decided to takeover.  Interesting protocol.

**Comment by vito** [comment]
@russmack ...huh? Not sure what "protocol" you're talking about, but I wouldn't be so quick to assume malice - they probably just didn't know about this PR. I'll ask.

In any case, this has nothing to do with Pivotal.

**Comment by shashwathi** [comment]
@russmack : I apologize. It was not my intention to take over this PR. I misread the date.

**Pull request closed without merging**